tools/gen_addon_readme.py

```diff
@@ -179,7 +182,7 @@ def gen_one_addon_readme(
             addon_dir, FRAGMENTS_DIR, fragment_name + '.rst',
         )
         if os.path.exists(fragment_filename):
-            with io.open(fragment_filename, 'rU', encoding='utf8') as f:
+            with io.open(fragment_filename, read_mode, encoding='utf8') as f:
```

Comment:
Since the U mode was deprecated since python 3.3 you can simply drop the read_mode ('r' is the default).
BTW, while you are at it, you can replace `io.open` with `open`.

Reply:
Sure, I wanted to keep the old compatibility, but you are right :smile:

Reply:
Done! :smile: